scripts/scil_mrds_select_number_of_tensors.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Fit the info from a NuFO map to the plausible MRDS solutions.
```

Comment:
Use the NUFO map information to select the plausible number of tensors in the MRDS solutions.